tests/test_transmission_matrix.py

```diff
@@ -73,6 +80,86 @@ def test_tmatrix_from_abcd_input_types(frequencies, A_list, A_np, A_FreqDat):
     _expect_error_abcd_same_type(A_np, A_FreqDat, A_np, A_np)
     _expect_error_abcd_same_type(A_FreqDat, A_np, A_np, A_np)
 
+def test_tmatrix_from_abcd_broadcasting(frequencies):
```

Comment:
this might be a good example for using parameters, see e.g. [here](https://github.com/pyfar/pyfar/blob/89f9ea1925756498ce751022fcab621139544c58/tests/test_coordinates_new.py#L16C14-L16C25). 
i would be also god practive to split the working and expected errors in separate tests (even we often missed it).
the output should also be tested.